recipes/qpdf/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.51.3"
+
+class PackageConan(ConanFile):
+    name = "qpdf"
+    description = "QPDF is a command-line tool and C++ library that performs content-preserving transformations on PDF files."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qpdf/qpdf"
+    topics = ("pdf")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_crypto": ["native","openssl"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_crypto": "native",
+    }
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "5",
+            "Visual Studio": "19.10",
+            "clang": "3.4",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        # https://qpdf.readthedocs.io/en/stable/installation.html#basic-dependencies
+        if self.options.with_crypto == "openssl":
+            self.requires("openssl/3.0.5")
+        self.requires("zlib/1.2.12")
+        self.requires("libjpeg-turbo/2.1.4")
+
+
+    def validate(self):
+        # validate the minimum cpp standard supported. For C++ projects only
```

Comment:
```suggestion
```
Keep only comments related to the project. For recipes, it's all documented already.